R/nhs_selectizeInput.R

```diff
@@ -1,11 +1,27 @@
-#' nhs_selectizeInput Function
+#' nhs_selectizeInput
 #'
-#' @importFrom shiny tagList
+#' NHS styled selectizeInput
+#'
+#' @param inputId Element id
+#' @param label Element label
+#' @param full_width Boolean, controls if full width or not
+#' @param ... Further named args passed to \code{shiny::selectInput}
+#'
+#' @return HTML
+#' @export
+#'
+#' @examples
+#' nhs_selectizeInput(
+#'   "fruit",
+#'   "Choose some fruit",
+#'   full_width = TRUE
+#' )
```

Comment:
Added some documentation - but I don't think this really does anything...possibly partly implemented and then forgotten?

Reply:
what do you mean?